gmail/Module.md

```diff
@@ -1,54 +1,30 @@
-Connects to Gmail from Ballerina.
+## Overview
+Ballerina Gmail Connector provides the capability to send, read and delete emails through the Gmail REST API. It also provides the ability to read, trash, untrash and delete threads, ability to get the Gmail profile and mailbox history, etc. The connector handles OAuth 2.0 authentication.
```

Comment:
1) send, read, and delete
2) we are repeating read. Lets remove it from the latter. 
3) trash, un-trash, and delete
4) threads, ability -> as well as the ability
5) The connector handles -> the connector supports

Reply:
The read mentioned in point 2 is required. That is not a repetitive one. First read is for messages and second read for threads. So we need to keep both. 
Other issues fixed by ddd90073069938a3660747b110c4643c53e57b4f